.github/workflows/yaml_check.yaml

```diff
@@ -0,0 +1,15 @@
+name: Lint YAML
+on: [pull_request]
+jobs:
+  build:
```

Comment:
This doesn't have to be "build". If we rather have a different label.

Reply:
Switched to `check`